engine/gamesys/src/gamesys/components/comp_camera.cpp

```diff
@@ -258,59 +288,26 @@ namespace dmGameSystem
         CameraComponent* camera = (CameraComponent*)*params.m_UserData;
         if ((dmDDF::Descriptor*)params.m_Message->m_Descriptor == dmGamesysDDF::SetCamera::m_DDFDescriptor)
         {
-            dmGamesysDDF::SetCamera* ddf = (dmGamesysDDF::SetCamera*)params.m_Message->m_Data;
-            camera->m_AspectRatio = ddf->m_AspectRatio;
-            camera->m_Fov = ddf->m_Fov;
-            camera->m_NearZ = ddf->m_NearZ;
-            camera->m_FarZ = ddf->m_FarZ;
-            camera->m_OrthographicProjection = ddf->m_OrthographicProjection;
-            camera->m_OrthographicZoom = ddf->m_OrthographicZoom;
+            dmRender::RenderContext* render_context = (dmRender::RenderContext*)params.m_Context;
+            dmRender::RenderCameraData camera_data = dmRender::GetRenderCameraData(render_context, camera->m_RenderCamera);
+
+            dmGamesysDDF::SetCamera* ddf         = (dmGamesysDDF::SetCamera*)params.m_Message->m_Data;
+            camera_data.m_AspectRatio            = ddf->m_AspectRatio;
+            camera_data.m_Fov                    = ddf->m_Fov;
+            camera_data.m_NearZ                  = ddf->m_NearZ;
+            camera_data.m_FarZ                   = ddf->m_FarZ;
+            camera_data.m_OrthographicProjection = ddf->m_OrthographicProjection;
+            camera_data.m_OrthographicZoom       = ddf->m_OrthographicZoom;
+
+            dmRender::SetRenderCameraData(render_context, camera->m_RenderCamera, camera_data);
```

Comment:
We could potentially return the pointer to the data to avoid calling get + set, but this is a bit cleaner imo